aerosandbox/library/power_solar.py

```diff
@@ -70,6 +99,52 @@ def incidence_angle_function(latitude, day_of_year, time, scattering=True):
     else:
         return cosine_factor * scattering_factor(elevation_angle)
 
+def incidence_angle_function_vert(latitude, day_of_year, time, scattering=True):
+    """
+    What is the fraction of insolation that a horizontal surface will receive as a function of sun position in the sky?
+    :param latitude: Latitude [degrees]
+    :param day_of_year: Julian day (1 == Jan. 1, 365 == Dec. 31)
+    :param time: Time since (local) solar noon [seconds]
+    :param scattering: Boolean: include scattering effects at very low angles?
+    """
+    # Old description:
+    # To first-order, this is true. In class, Kevin Uleck claimed that you have higher-than-cosine losses at extreme angles,
+    # since you get reflection losses. However, an experiment by Sharma appears to not reproduce this finding, showing only a
+    # 0.4-percentage-point drop in cell efficiency from 0 to 60 degrees. So, for now, we'll just say it's a cosine loss.
+    # Sharma: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611928/
+
+    elevation_angle = solar_elevation_angle(latitude, day_of_year, time)
+    theta = 90 - elevation_angle - 90  # Angle between panel normal and the sun, in degrees
+
+    cosine_factor = np.cosd(theta)
+
+    if not scattering:
+        return cosine_factor
+    else:
+        return cosine_factor * scattering_factor_vert(elevation_angle)
+
+def incidence_angle_function_new(latitude, day_of_year, time, panel_heading, panel_angle, scattering=True):
+    """
+    This website will be useful for accounting for direction of the vertical surface
+    https://www.pveducation.org/pvcdrom/properties-of-sunlight/arbitrary-orientation-and-tilt
+    :param latitude: Latitude [degrees]
+    :param day_of_year: Julian day (1 == Jan. 1, 365 == Dec. 31)
+    :param time: Time since (local) solar noon [seconds]
+    :param panel_heading: the directionality of the solar panel (0 degrees if pointing North and 180 if South)
```

Comment:
We may want to rename this `panel_azimuth_angle` for consistency with solar nomenclature?